scripts/check_license

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Checks that source files have the Apache License header.
+# Automatically skips generated files.
+set -eu
```

Comment:
Could you source `scripts/common` here like we do in other scripts?

Reply:
Done.